core/chains/evm/gas/cmd/arbgas/main.go

```diff
@@ -45,14 +46,17 @@ func printGetLegacyGas(ctx context.Context, e gas.EvmEstimator, calldata []byte,
 }
 
 const max = 50_000_000
+const bumpPercent = 20
+
+var bumpMin = assets.NewWei(big.NewInt(5_000_000_000))
```

Comment:
How did we come up with this number? This seems very high for Arbitrum.

Reply:
Since the Arbitrum configs don't have `BumpPercent` or `BumpMin` set I set these using the ones in the [fallback.toml](https://github.com/smartcontractkit/chainlink/blob/d7fe683812367a8d32e6feae0436044c54282c9a/core/chains/evm/config/toml/defaults/fallback.toml#L36-L37). If this seems wrong, I could run it by the Ship pod. Although I don't think these values will be used. The [script](https://github.com/smartcontractkit/chainlink/blob/d7fe683812367a8d32e6feae0436044c54282c9a/core/chains/evm/gas/cmd/arbgas/main.go#L31-L33) is only calling `GetLegacyGas` on the estimator that we're creating with this config which doesn't add the buffer.

Reply:
I think for the reader it would be better to either remove them completely or use some values that make sense based on the Arbitrum configs.

Reply:
You're right. I've removed them so they don't cause any confusion